### DISCUSSION TITLE
High contrast color mode

### DISCUSSION THREAD

**Discussion by jannick-ux** [SchwarzIT/onyx#1945] [Ideas]
A high contrast theme could be introduced to make the applications even more accessible for people with visual impairment.
Should be defined, if we need one for every single brand including light and dark equivalent (high quantity) or if there is just one dedicated high contrast theme for dark and one for light mode (no dependencies to brands). That would lead to much less theming files

**Comment by mj-hof** [comment]
Still a good idea to offer the high contrast mode. Basically, accessibility is covered with all our provided themes.
